deploy/kubernetes/images/prow-gke-release-staging-rc/image.yaml

```diff
@@ -3,29 +3,29 @@ kind: ImageTagTransformer
 metadata:
   name: imagetag-csi-provisioner-prow-rc
 imageTag:
-  name: gke.gcr.io/csi-provisioner
-  newName: gcr.io/gke-release-staging/csi-provisioner
-  newTag: "v2.0.3-gke.0"
+  name: k8s.gcr.io/sig-storage/csi-provisioner
+  newName: gcr.io/k8s-staging-sig-storage/csi-provisioner
```

Comment:
staging-rc could remain on k8s.gcr.io/sig-storage since it's still using released images

Reply:
Do we not want to push it to use the latest available released staging image?

Reply:
For pd driver image, I think using the staging repo makes sense.

But I'm not sure about CSI sidecars. We don't use internal pdcsi tests as an indicator for OSS CSI sidecar releases.

Reply:
Good point. We don't need to test unreleased versions of sidecars in this repo.